mp/src/gameui/OptionsSubVideo.cpp

```diff
@@ -429,8 +415,8 @@ class COptionsSubVideoAdvancedDlg : public vgui::Frame
 
 		m_pDXLevel->SetEnabled(false);
 
-		m_pColorCorrection->SetEnabled( mat_dxlevel.GetInt() >= 90 );
-		m_pMotionBlur->SetEnabled( mat_dxlevel.GetInt() >= 90 );
+		m_pColorCorrection->SetEnabled(true);
+		m_pMotionBlur->SetEnabled(true);
```

Comment:
Do these need to be changed? On the off chance that we do allow different dx levels in the future this would become a problem. I'd recommend just leaving it as-is.

Reply:
Since `mat_dxlevel` is essentially removed from the game by the changes in cdll_client_int.cpp, referencing it here would throw errors in the console about not being able to find the cvar. I could add the ConVarRef for it back despite the fact it's removed though. 

Would just mean that the console would throw errors every time you enter/exit that menu until we'd add `mat_dxlevel` back.

Reply:
Didn't notice it was entirely removed. Just leave it as-is since that's the case.

Reply:
Do these need to be set to true here? Are they ever disabled by anything else?

Reply:
As far as I'm aware, these are only ever disabled by being below dxlevel 90.

Reply:
So since it's now impossible to go below dxlevel 90, shouldn't these two lines be removed?

Reply:
Fixed.